devicetypes/Orvibo/Orvibo-Contact-Sensor.src/Orvibo-Contact-Sensor.groovy

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
file location is wrong. It should be in devicetypes/smartthings/Orvibo-Contact-Sensor.src/Orvibo-Contact-Sensor.groovy